css-pseudo-4/Overview.bs

```diff
@@ -519,15 +519,56 @@ Selecting Highlighted Content: the ''::selection'',  ''::target-text'', ''::spel
       css/css-pseudo/grammar-error-002-manual.html
       css/css-pseudo/grammar-error-003-manual.html
       </wpt>
+
+    <dt><dfn>::highlight</dfn>
+    <dd>
+      The ''::highlight'' pseudo-element represents a portion of text contained in one or more
+      [=ranges=] associated with a [=custom highlight=] (see [[CSS-HIGHLIGHT-API-1]]).
```

Comment:
1. Since ::highlight() is a functional pseudo-element, it need parentheses.
2. Mention that each named highlight is considered a different type of highlight; this hooks into the definitions further down.
3. Let's make the [css-highlight-api-1] reference more definitive.

```suggestion
    <dt><dfn>::highlight()</dfn>
    <dd>
      The ''::highlight()'' [=functional pseudo-element=] represents content associated with [=custom highlights=].
      Each [=custom highlight name=] represents a different type of highlight.
      See [[CSS-HIGHLIGHT-API-1]] for details.
```